rebasehelper/application.py

```diff
@@ -576,11 +577,16 @@ def build_packages(self):
                 build_dict['binary_package_build_error'] = six.text_type(e)
                 results_store.set_build_data(version, build_dict)
 
-                if e.logfile is None:
+                if e.logfile is None and e.return_code is None:
                     msg = 'Building {} RPM packages failed; see logs in {} for more information'.format(
                         version, os.path.join(results_dir, 'RPM')
                     )
                 else:
+                    if e.logfile is None:
+                        # koji was used for build therefore only status code is set
+                        e.logfile = MockBuildTool.get_mock_logfile_path(e.return_code,
+                                                                        os.path.join(results_dir, 'RPM'))
+
```

Comment:
I think it would make more sense to write this part of code like this:
```python
if e.return_code:
    # set e.logfile
if e.logfile:
    ...
else:
    ...
```